kolibri/core/assets/src/views/side-nav/index.vue

```diff
@@ -204,24 +187,56 @@
             href: '/user',
           });
         }
+        options.push({ type: 'divider' });
         return options;
       },
     },
-    components: {
-      uiMenu,
-      uiIcon,
-      uiIconButton,
-      logo,
+    watch: {
+      navShown(isShown) {
+        this.$nextTick(() => {
+          if (isShown) {
+            window.addEventListener('focus', this.containFocus, true);
+            this.previouslyFocusedElement = document.activeElement;
+            this.$refs.toggleButton.$el.focus();
+          } else {
+            window.removeEventListener('focus', this.containFocus, true);
+            this.previouslyFocusedElement.focus();
+          }
+        });
+      },
+    },
+    methods: {
+      navigate(option) {
+        if (option.href) {
+          window.location.href = option.href;
+        } else if (option.action) {
+          option.action();
+        }
+      },
+      toggleNav() {
+        this.$emit('toggleSideNav');
+      },
+      pageIsActive(pageName) {
+        return this.topLevelPageName === pageName;
+      },
+      containFocus(event) {
+        if (event.target === window) {
+          return;
+        }
+        if (!this.$refs.sideNav.contains(event.target)) {
+          this.$refs.toggleButton.$el.focus();
+        }
+      },
     },
     vuex: {
-      actions: { signOut: actions.kolibriLogout },
+      actions: { signOut: kolibriLogout },
       getters: {
         session: state => state.core.session,
-        isUserLoggedIn: getters.isUserLoggedIn,
-        isSuperuser: getters.isSuperuser,
-        isAdmin: getters.isAdmin,
-        isCoach: getters.isCoach,
-        canManageContent: getters.canManageContent,
+        isUserLoggedIn: isUserLoggedIn,
```

Comment:
You can drop the values here and use enhanced object syntax:
```
{
...
isUserLoggedIn,
isSuperuser,
...
}
```

Reply:
hopefully once we add eslint back in, this will be enforced again by The System

https://github.com/learningequality/kolibri/issues/2216